### ISSUE TITLE
Model Unpickling error in Google Colab

### ISSUE THREAD

**Issue by SouravSarkar865** [wgcban/ChangeFormer#66]
I am getting the following error when I am trying to load the model in colab after downloading it:
UnpicklingError: invalid load key, '<'.
I have tried re-downloading the model and still got the same error.
I also tried downgrading to previous version of torch but it did not downgrade below 1.11.0 while the requirement mentioned in the Readme file is 1.10.1, so I tried loading the model in torch version 2.0 as well as 1.11 but it kept giving the same error.

**Comment by wgcban** [comment]
@SouravSarkar865 this is mainly due to PyTorch version mismatch. Please check your version of PyTorch matches with the version in the `requirements.txt`.  Thanks.

**Comment by SouravSarkar865** [comment]
Thank you. I had to change the version of some of the dependencies in the colab environment to match the requirements and was able to load the model.
Now that I can load the model into colab directly, can you please tell how I can I use the model to find the change between two random images from my colab environment folder.

**Comment by wgcban** [comment]
@SouravSarkar865 follow the demo `demo_LEVIR.py`: https://github.com/wgcban/ChangeFormer#speech_balloon-quick-start-on-levir-dataset

**Comment by SouravSarkar865** [comment]
Thank you. Will do the needful.

**Comment by SouravSarkar865** [comment]
Hello.
I followed the instructions, copied the repository, downloaded the model and kept in the folder mentioned  in the Readme file. I gave the checkpoint_dir path and vis path wherever it was necessary in the scipts and notebooks copying them directly from my colab environment and thoroughly rechecked them to make sure they were correct. 

Still, when I ran the demo_LEVIR.py file, I keep got the following error:

FileNotFoundError: no such checkpoint pretrained_changeformer.pt
